src/screens/TransactionDetail.js

```diff
@@ -82,6 +83,32 @@ class TransactionDetail extends React.Component {
     });
   }
 
+  /**
+   * Get transaction metadata from explorer service
+   */
+  updateTxMetadata = (id) => {
+    metadataApi.getDag(id).then((data) => {
+      if (data) {
+        this.setState((oldState) => {
+          return {
+            transaction: {
+              ...oldState.transaction,
+              meta: data
```

Comment:
This is getting a bit confusing here. We already have in the state `this.state.meta` which is the metadata that we receive from the full node (the transaction has its own metadata).

I'm not sure about the best name, maybe call `customMeta`? I don't know, what do you think?

Reply:
I was confused as well, the only thing i'm certain is that we don't want to merge metadata from the fullnode and from the explorer-service api. I included on the tx because that's what we do for the token but I think we should change the name